modules/apps/asset/asset-info-impl/src/main/java/com/liferay/asset/info/internal/item/provider/AssetEntryInfoItemFieldSetProviderImpl.java

```diff
@@ -257,11 +257,21 @@ private Set<AssetVocabulary> _getNoninternalAssetVocabularies(
 	private List<AssetVocabulary> _getNoninternalAssetVocabularies(
 		String itemClassName, long itemClassTypeId, long scopeGroupId) {
 
+		long[] groupsIds;
+
+		try {
+			groupsIds =
+				SiteConnectedGroupGroupProviderUtil.
```

Comment:
Why doyou use the Util instead of inject the reference?

Reply:
Thanks Roberto, I've made that update.